File: CineBox/CineBox.Model/ViewModels/Review.cs
﻿using System;
namespace CineBox.Model.ViewModels
{
	public partial class Review
	{
        public int Id { get; set; }

        public int UserId { get; set; }

        public int MovieId { get; set; }

        public int Rating { get; set; }

        public string Comment { get; set; } = null!;
    }
}

File: CineBox/UI/cinebox_mobile/lib/models/Review/review.dart
import 'package:json_annotation/json_annotation.dart';

part 'review.g.dart';

@JsonSerializable()
class Review {
  int? id;
  int? userId;
  int? movieId;
  double? rating;
  String? comment;

  Review();


  factory Review.fromJson(Map<String, dynamic> json) => _$ReviewFromJson(json);


  Map<String, dynamic> toJson() => _$ReviewToJson(this);
}